client/state/selectors/get-allow-restore.js

```diff
@@ -22,12 +22,12 @@ export default createSelector(
 			! isMultiSite &&
 			'active' === rewind.state &&
 			! ( 'queued' === restoreStatus || 'running' === restoreStatus ) &&
-			includes( siteCapabilities, 'restore' )
+			siteHasRestore
 		);
 	},
 	( state, siteId ) => {
 		return [
-			getRewindCapabilities( state, siteId ),
+			siteHasFeature( state, siteId, WPCOM_FEATURES_BACKUPS_RESTORE ),
```

Comment:
Am I doing this right? Not sure if that call needs to be duplicated in the dependents here?